images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -0,0 +1,80 @@
+################################################################################
+##  File:  Install-Msys2.ps1
+##  Desc:  Install Msys2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$env:orig_path = $env:PATH
+$env:git_path  = "C:\Program Files\Git"
+
+# get info from https://sourceforge.net/projects/msys2/files/Base/x86_64/
+$msy2_uri  = "http://repo.msys2.org/distrib/x86_64/msys2-base-x86_64-20190524.tar.xz"
```

Comment:
nitpick: $msys2_uri. Also, could ypu please adjust variables naming to our common style that is used in other installers files and get rid of underlines?

Reply:
Done